src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -7,3 +7,37 @@
       first
       :balance
       js/parseInt))
+
+(def ^:const supported-collectible-types
+  #{"image/jpeg"
+    "image/gif"
+    "image/bmp"
+    "image/png"
+    "image/webp"})
+
```

Comment:
cc @vkjr @ulisesmac - this list is good for now?

Reply:
I think we need to manually explore if there are more supported types other than these.

But I'm pretty sure these work.

Reply:
Wondering if we can add a log message when unsupported collectible type encountered. Just to make sure we will notice it later.

Reply:
Done